base/setup/usetup/lang/pl-PL.h

```diff
@@ -1815,19 +1816,22 @@ static MUI_ENTRY plPLRegistryEntries[] =
         4,
         3,
         " Instalator ReactOS " KERNEL_VERSION_STR " ",
-        TEXT_STYLE_UNDERLINE
+        TEXT_STYLE_UNDERLINE,
+        TEXT_ID_STATIC
```

Comment:
Do you keep these style changes in sync with other translations?

Reply:
yes, these lines are missing in [pl-PL]. Compared 2 files with Winmerge ... ;)